src/client.h

```diff
@@ -30,11 +30,13 @@ extern ADDON::CHelper_libXBMC_addon*  XBMC;
 extern CHelper_libXBMC_pvr*           PVR;
 extern CHelper_libXBMC_codec*         CODEC;
 
-#define DEFAULT_HOST             "127.0.0.1"
-#define DEFAULT_HTTP_PORT        9981
-#define DEFAULT_HTSP_PORT        9982
-#define DEFAULT_CONNECT_TIMEOUT  10
-#define DEFAULT_RESPONSE_TIMEOUT 5
+#define DEFAULT_HOST                "127.0.0.1"
+#define DEFAULT_HTTP_PORT           9981
+#define DEFAULT_HTSP_PORT           9982
+#define DEFAULT_CONNECT_TIMEOUT     10
+#define DEFAULT_RESPONSE_TIMEOUT    5
+#define DEFAULT_PRETUNERS           1
```

Comment:
Perhaps we should change this setting to TOTAL_TUNERS instead since that's what it actually represents, right? I have a feeling someone is going to be confused and set this value to zero because they only have one tuner.